src/mgmt/srv.c

```diff
@@ -577,16 +574,16 @@ process_drpc_request(Drpc__Call *drpc_req, Drpc__Response *drpc_resp)
 		if (body == NULL) {
 			drpc_resp->status = DRPC__STATUS__FAILURE;
 			D_ERROR("Failed to allocate drpc response body\n");
+		} else {
+			mgmt__create_pool_resp__pack(create_pool_resp, body);
+			drpc_resp->body.len = len;
+			drpc_resp->body.data = body;
+		}
+		/** check for '\0' which is a static allocation from protobuf */
+		if (create_pool_resp->svcreps && create_pool_resp->svcreps[0] != '\0')
```

Comment:
Almost feel like it would be nice to have a helper function or macro for this, since we're already checking it twice. If this is protobuf default behavior, seems likely to come up again.